src/verkle/keyvals/mod.rs

```diff
@@ -0,0 +1,50 @@
+// https://github.com/gballet/verkle-block-sample
+
+use rlp::{Decodable, DecoderError, Rlp};
+use std::convert::TryInto;
+
+#[derive(Debug)]
+pub struct Tuple(Vec<u8>, Vec<u8>);
+
+impl Decodable for Tuple {
+    fn decode(rlp: &Rlp<'_>) -> std::result::Result<Self, DecoderError> {
+        Ok(Tuple(rlp.val_at::<Vec<u8>>(0)?, rlp.val_at::<Vec<u8>>(1)?))
+    }
+}
+
+impl TryInto<([u8; 32], Option<[u8; 32]>)> for Tuple {
+    type Error = String;
```

Comment:
you don't have any errors, so you can make it `type Error = ();` 
but if you don't have any errors, you can just `impl Into`, not `TryInto`.
also it's better to implement `From`: https://doc.rust-lang.org/std/convert/trait.From.html
> One should always prefer implementing From over [Into](https://doc.rust-lang.org/std/convert/trait.Into.html) because implementing From automatically provides one with an implementation of [Into](https://doc.rust-lang.org/std/convert/trait.Into.html) thanks to the blanket implementation in the standard library.